app/src/main/java/com/android/sample/model/recipe/RecipeRepository.kt

```diff
@@ -0,0 +1,6 @@
+package com.android.sample.model.recipe
+
+interface RecipeRepository {
+
+  fun random(nbOfElements: Long, onSuccess: (List<Recipe>) -> Unit, onFailure: (Exception) -> Unit)
```

Comment:
Like for the data class Recipe, I think it should be helpful to have a description for each parameter as well as having a description of the function in order to know what it does.